feature/login/src/main/java/com/simprints/feature/login/screens/form/LoginFormFragment.kt

```diff
@@ -50,6 +51,17 @@ internal class LoginFormFragment : Fragment(R.layout.fragment_login_form) {
     private val binding by viewBinding(FragmentLoginFormBinding::bind)
     private val viewModel by viewModels<LoginFormViewModel>()
 
+    private lateinit var checkForPlayServicesResultLauncher: ActivityResultLauncher<IntentSenderRequest>
+    init {
```

Comment:
You can skip the init block and just declare it inline. It would be identical in execution, but a bit less disjointed.

Reply:
There's a reference to `checkForPlayServicesResultLauncher` inside `registerForActivityResult(ActivityResultContracts.StartIntentSenderForResult()) { ... }` block, so an inlined declaration wouldn't work.